content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,143 @@ aliases:
-[service subsets](/docs/concepts/traffic-management/#service-subsets)
-in a virtual service to configure the [routing rules](/docs/concepts/traffic-management/#routing-rules)
-for multiple versions of a service.
-
-Service subsets are used to label all instances that correspond to a specific
-version of a service.
-Before you configure routing rules, the Envoy proxies use round-robin load
-balancing across all service instances, regardless of their subset. After you
-configure routing rules for traffic to reach specific subsets, the Envoy
-proxies route traffic to the subset according to the rule but again use
-round-robin to route traffic across the instances of each subset.
-
-This configuration method provides the following advantages:
-
-- Decouples the application code from the evolution of the application's dependent services.
-- Provides monitoring benefits.
-For details, see [Mixer policies and telemetry](/docs/reference/config/policy-and-telemetry/).
-
-For example, in A/B testing we often want to configure traffic routes based on
-percentages. With Istio, you can use a virtual service to specify a routing
-rule that sends 25% of requests to instances in the `v2` subset, and sends the
-remaining 75% of requests to instances in the `v1` subset. The following
-configuration accomplishes our example for the `reviews` service.
+Istio’s traffic routing rules let you easily control the flow
+of traffic and API calls between services. Istio simplifies configuration of
+service-level properties like circuit breakers, timeouts, and retries, and makes
+it easy to set up important tasks like A/B testing, canary rollouts, and staged
+rollouts with percentage-based traffic splits. It also provides out-of-box
+failure recovery features, which, combined with Istio’s
+[observability](/docs/concepts/observability/) features, helps make your application
+more robust against failures of dependent services or the network.
+
+Istio’s traffic management model relies on the {{< gloss >}}Envoy{{</ gloss >}}
+proxies that are deployed along with your services. All traffic that your mesh
+services send and receive ({{< gloss >}}data plane{{</ gloss >}} traffic) is proxied through Envoy, making
+it easy to direct and control traffic around your mesh without making any
+changes to your services.
+
+If you’re interested in the details of how the features described in this guide
+work, you can find out more about Istio’s traffic management architecture in the
+[Architecture](#architecture) section at the end of this document. The rest of
+this guide introduces Istio’s traffic management features.
+
+## Introducing Istio Traffic Management
+
+In order to direct traffic around your mesh, Istio needs to know where all your
+endpoints are, and which services they belong to. To populate its own
+{{< gloss >}}service registry{{</ gloss >}}, Istio connects to a service
+discovery system. For example, if you've installed Istio on a Kubernetes cluster,
```

Comment:
remove For example, and start the sentence with If.

Installed Istio on Kubernetes (remove a).

Reply:
I think we should leave in "For example", otherwise (per Srinath's concerns above) it's not obvious that this is just one of the platforms where you could install Istio. Also removing "cluster" makes the sentence a bit odd because we're talking specifically about a cluster: "the services and endpoints on that cluster".

Reply:
Don't change this. P.S. you mean Shriram (no Srinath).

Reply:
Gah, of course I do. Sorry Shriram!